### PR TITLE
[#6091] academy/block_academy_challenge: add the progress circle with fill an…

### PR DESCRIPTION
**Pull request by phillimorland**
…d step number

### CODE REVIEW

**Review by fuzzylogic2000 — COMMENTED** [review]
It does look awesome! 
I have two questions, though.

File: apps/core/static/scss/_progress-circle.scss [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    margin: 8px;
+    border-radius: 50%;
+    background: $body-bg;
+    font-size: 1rem;
+    text-align: center;
+    font-family: $font-family-monospace;
+}
+
+@keyframes turn-in {
+    5% {
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply apps/core/static/scss/_progress-circle.scss:63]: it's for the animation of the filling up of the circles so each frame is it filling

File: apps/academy/templates/academy/blocks/block_academy_challenge.html [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
-    {% for link in value.links %}
-            {% include "academy/includes/academy_challenge_list_item.html" with link=link.challenge_step_academy_links %}
-            {% include "academy/includes/academy_challenge_list_item.html" with link=link.challenge_step_external_links %}
-    {% endfor %}
+        {% for link in value.links %}
+        {% include "academy/includes/academy_challenge_list_item.html" with link=link.challenge_step_academy_links %}
+        {% include "academy/includes/academy_challenge_list_item.html" with link=link.challenge_step_external_links %}
+        {% endfor %}
+        <div class="col-lg-3 progress-circle__container">
+          {{max}}
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply apps/academy/templates/academy/blocks/block_academy_challenge.html:19]: yes :p

**Comment by phillimorland** [comment]
@fuzzylogic2000 removed migration as will now review your pr

**Comment by fuzzylogic2000** [comment]
> @fuzzylogic2000 removed migration as will now review your pr

Sorry! Somehow, I overlooked that here were migrations as well. Without the leftover from trying stuff, you can merge as soon as you added them back.

**Comment by phillimorland** [comment]
@fuzzylogic2000 cool i'll just wait till it's passed CI

**Pull request merged**